src/mpid/ch4/netmod/ofi/ofi_impl.h

```diff
@@ -327,7 +327,7 @@ int MPIDI_OFI_control_handler(int handler_id, void *am_hdr,
                               MPIDIG_am_target_cmpl_cb * target_cmpl_cb, MPIR_Request ** req);
 int MPIDI_OFI_control_dispatch(void *buf);
 void MPIDI_OFI_index_datatypes(void);
-void MPIDI_OFI_index_allocator_create(void **_indexmap, int start, MPL_memory_class class);
```

Comment:
Why is this no longer needed? Please explain in the commit message.

Reply:
I am not sure why it was designed like this in the first place. My only guess is because we have two allocators and we want to avoid id collisions from the two allocators. But, that will be weird since we only offset the output of these two allocators by one.

Reply:
The simplified implementation no longer needs that since we are allocating from the same allocator which won't have any collision.

Reply:
I was hoping some explanation would get added to the commit message regarding this change. Another option would be to squash this with the other simplify commit. If that is not trivial because of rebase conflicts, then I'll relent in the interest of getting this merged.